src/context.rs

```diff
@@ -1167,11 +1167,13 @@ impl BlockOffset {
   /// Offset of the top-left pixel of this block.
   pub fn plane_offset(&self, plane: &PlaneConfig) -> PlaneOffset {
     let po = self.sb_offset().plane_offset(plane);
+
     let x_offset = self.x & LOCAL_BLOCK_MASK;
     let y_offset = self.y & LOCAL_BLOCK_MASK;
+
     PlaneOffset {
-      x: po.x + (x_offset << BLOCK_TO_PLANE_SHIFT),
-      y: po.y + (y_offset << BLOCK_TO_PLANE_SHIFT)
+        x: po.x + (x_offset >> plane.xdec << BLOCK_TO_PLANE_SHIFT),
```

Comment:
Shouldn't this be << (BLOCK_TO_PLANE_SHIFT - plane.xdec), similar to the other plane_offset function ?

Reply:
@smarter, hey more essence of the reason that I think I missed when I originally replied Aug, 2018 is now here! https://github.com/xiph/rav1e/issues/1062#issuecomment-469812492